src/runtime/virtcontainers/hypervisor_amd64.go

```diff
@@ -21,7 +21,7 @@ func availableGuestProtection() (guestProtection, error) {
 	}
 	// SEV is supported and enabled when the kvm module `sev` parameter is set to `1`
 	if _, err := os.Stat(sevKvmParameterPath); err == nil {
-		if c, err := ioutil.ReadFile(sevKvmParameterPath); err == nil && len(c) > 0 && c[0] == '1' {
+		if c, err := ioutil.ReadFile(sevKvmParameterPath); err == nil && len(c) > 0 && (c[0] == '1' || c[0] == 'Y') {
```

Comment:
We should probably normalize the string here. I have seen different case behavior among different distributions of Linux.